package.json

```
@@ -26,16 +26,17 @@
     "migrate:ts": "npm-run-all build:cache migrate:up:dist",
     "migrate:up": "yarn db-migrate -m migrations up",
     "migrate:up:dist": "yarn db-migrate -m dist up",
-    "mysql": "docker compose exec mysql serlo-mysql",
-    "mysql:ci": "docker compose exec mysql serlo-mysql",
+    "mysql": "yarn mysql:cli",
+    "mysql:cli": "docker compose exec mysql serlo-mysql",
```

I can't remember why, but back then it was necessary to split between a script for ci and another for the local development. But if it is now outdated and both script do the same same, we can remove mysql:cli/ci and use just one.

```suggestion
    "mysql": "docker compose exec mysql serlo-mysql",
```
Of course, everything else would have to be changed to `yarn mysql`

---

I deleted all `:cli` versions of the command since they aren't needed any more -> Ready for review